### PR TITLE
Evan Poole

### PR DESCRIPTION
**Pull request by thetechtakeover**
Added Evan Poole's information html file and stylesheet

### CODE REVIEW

File: students/style.css [hunk 1]
```diff
@@ -0,0 +1,52 @@
+* {
```
Comment (nellarro) [inline students/style.css:1]: 💭 Love this file. I'd recommend making sure that the spacing between all of the CSS looks consistent though:

```
 .css {
       property: value;
       property: value;
   }
```

File: students/style.css [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+p{
+    margin: 20px 0px ;
+}
+
+ul, ol {
+    margin-left: 1em;
+    text-align: left;
+} 
+
+a{
```
Comment (nellarro) [inline students/style.css:38]: 🙃 watch your spacing, here

**Review by nellarro — APPROVED** [review]
Looks great! left a few comments :)

**Comment by thetechtakeover** [comment]
> 
> 
> Looks great! left a few comments :)

I'm so sorry for the late reply. I checked almost everyday to see if my work was accepted and did not understand what i was looking for lol and I did not realized you left a comment. I did not know how to view the thread :(

**Pull request merged**